cli/src/main/java/org/eclipse/hono/cli/AbstractCliClient.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2016, 2018 Contributors to the Eclipse Foundation
+ * Copyright (c) 2016, 2018, 2020 Contributors to the Eclipse Foundation
```

Comment:
This should specify the start and end of the interval during which this file was created and modified. Sorry for not having been specific enough about it. So, this should read
```
 * Copyright (c) 2016, 2020 Contributors to the Eclipse Foundation
```